docs/static/logging.asciidoc

```diff
@@ -48,6 +48,67 @@ The logger is usually identified by a Java class name, such as
 path as in `org.logstash.dissect`.  For Ruby classes, like `LogStash::Outputs::Elasticsearch`,
 the logger name is obtained by lowercasing the full class name and replacing double colons with a single dot.
 
+It's strongly recommended to use the default log4j configuration that's shipped with {ls}, however the
+following section describe in detail how the rolling strategy works, so you have more control on these settings.
```

Comment:
```suggestion
NOTE: Consider using the default log4j configuration that is shipped with {ls}, as it is configured to work well for most deployments.  
The next section describes how the rolling strategy works in case you need to make adjustments.
```